examples/fib_asyncio.py

```diff
@@ -9,7 +9,7 @@
 L = s.sink_to_list()                    # store result in a list
 
 s.rate_limit(0.5).sink(source.emit)         # pipe output back to input
-s.rate_limit(1.0).sink(lambda x: print(L))  # print state of L every second
+s.timed_window(1.0).sink(lambda x: print(x))
```

Comment:
Did you mean to change this in the PR?

Reply:
Yes. It changes the functionality a bit but now this example tests the rate_limit() and timed_window() methods. I've been using this to test the other changes I've been making in this PR.